mozilla/activity-stream
# Preparing the Activity Stream release 1.2.0

Hi all,

We are now preparing the release of A-S 1.2.0. The add-on has been made available at [pre-release channel](https://moz-activity-streams-prerelease.s3.amazonaws.com/dist/latest.html). Please install it, test it, and leave the feedback, bug reports, and other inputs in this thread.

Highlights&Changelog:
* [new] new A/B experiment to test uncombined/combined frecency scores of deduped top sites
* [new] add various new telemetry pings
  * bad states (image missing, not enough highlights, and showing hang-on spinner)
  * highlights number
  * preference change event
* [fix] refresh highlights upon the sync's update
* [fix] always show the recently bookmarked sites regardless of their metadata
* [fix] various of UI/UX changes
* [dev] introduce the feed concept for TopSites, Highlights, Metadata, and Search
* [dev] remove the default highlight tiles
* [dev] deprecate the recommendation module and related prefs

# Release checklist
- QA
- Product
- Engineering

Please use the `release/block` or `release/uplift` to label the bug reports as well as other release related issues. Once you finish the review, please check off the corresponding item in the above checklist.

*Note*: checking off the item may not work, you can either modify it in place (i.e modify `[ ]` to `[x]`), or just drop a good old `r+` comment below.

*Note*: release highlights and the changelog will be attached shortly.

Thanks


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/activity-stream/1939)
